srcpkgs/cryptsetup/template

```diff
@@ -16,8 +16,8 @@ license="GPL-2.0-or-later"
 homepage="https://gitlab.com/cryptsetup/cryptsetup"
 changelog="https://gitlab.com/cryptsetup/cryptsetup/raw/master/docs/v${version}-ReleaseNotes"
 distfiles="${KERNEL_SITE}/utils/cryptsetup/v${version%.*}/${pkgname}-${version}.tar.xz"
-checksum=fc0df945188172264ec5bf1d0bda08264fadc8a3f856d47eba91f31fe354b507
-make_check=extended
+checksum=9184a6ebbd9ce7eb211152e7f741a6c82f2d1cc0e24a84ec9c52939eee0f0542
+make_check=ci-skip # tests depend on acessing /dev/mapper/control fails on CI
```

Comment:
some tests fail in local builds too

Reply:
Replied below, but if you want I can add the make_check=no because they require root to run and are skipped locally.

=======================
All 9 tests passed
(22 tests were not run)
=======================

Reply:
```
$ grep CHECK etc/conf
XBPS_CHECK_PKGS=full

$ ./xbps-src binary-bootstrap x86_64-musl && ./xbps-src pkg cryptsetup
...
FAIL: luks2-reencryption-mangle-test
WARNING: You must be root to run this test, test skipped.
SKIP: verity-compat-test
WARNING: You must be root to run this test, test skipped.
SKIP: integrity-compat-test
WARNING: You must be root to run this test, test skipped.
SKIP: ssh-test-plugin
=======================
2 of 10 tests failed
(21 tests were not run)
=======================
make[2]: *** [Makefile:900: check-TESTS] Error 1
```

Reply:
Reported upstream (https://gitlab.com/cryptsetup/cryptsetup/-/issues/781)

Are you okay with adding make_check=no if libc is musl on the template ?